tck/src/main/java/org/reactivestreams/tck/support/PublisherVerificationRules.java

```diff
@@ -297,6 +297,18 @@
    */
   void optional_spec111_maySupportMultiSubscribe() throws Throwable;
   /**
+   * Ask for a single-element {@code Publisher} and subscribes to it twice.
+   * Each {@code Subscriber} requests for 1 element and checks if onNext or onComplete signals was received.
```

Comment:
@ktoso i think, that test expect such behavior...
It publisher can emit 0 elements - both subscribers will receive onComplete signal on demand.
If publisher can emit 1 element:
a) multicast - both subscribers will receive onNext signal on demand.
b) unicast - one subscriber will receive onNext signal (and optinal onComplete - impl dependent - i.e. lazy/eager publisher), and another will receive onComplete on demand (publisher already in terminal state)
If publisher can emit n > 1 elements, both subscribers will receive onNext signal on demand.

So, test verifies that signals was received, but not it's type.